ndless-sdk/tools/zehn_loader/Makefile

```diff
@@ -1,18 +1,28 @@
 GXX := "$(shell (which arm-elf-g++ arm-none-eabi-g++ arm-linux-gnueabi-g++ | head -1) 2>/dev/null)"
 OBJCOPY := "$(shell (which arm-elf-objcopy arm-none-eabi-objcopy arm-linux-gnueabi-objcopy | head -1) 2>/dev/null)"
+CHOST := $(shell (echo $(shell basename $(GXX)) | sed 's/-g++$$//'))
 
-GXXFLAGS := -g -Os -Wall -Wextra -march=armv5te -fPIE -std=c++11 -fno-rtti -fno-exceptions -Wl,-Tldscript -nostdlib -nostartfiles -I ../../include -D _LDBL_EQ_DBL
+GXXFLAGS := -g -Os -Wall -Wextra -march=armv5te -fPIE -std=c++11 -fno-rtti -fno-exceptions -Wl,-Tldscript -Wl,--gc-sections -nostdlib -nostartfiles -I ../../include
+ZLIB_CFLAGS := -Os -fPIE -ffunction-sections -fdata-sections -DNO_GZIP -c
```

Comment:
A workaround for the zlib configure script to work correctly before the rest of the SDK is built.